h11/_state.py

```diff
@@ -197,7 +197,7 @@
 }
 
 
-class ConnectionState(object):
+class ConnectionState:
```

Comment:
Should we use `__slots__` for this object as well since it's accessed frequently?

Reply:
I'm in favor of adding slots to everything personally. I'll do it in a separate PR after some of the others are settled.